buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/ArtifactTaskName.kt

```diff
@@ -35,5 +35,6 @@ package io.spine.internal.gradle.publish
 internal enum class ArtifactTaskName {
     sourceJar,
     testOutputJar,
-    javadocJar;
+    javadocJar,
```

Comment:
Another thing here is that today we are going to have [one more PR merged to `config`](https://github.com/SpineEventEngine/config/pull/335). In its scope, this code has been eliminated (probably, for good).

Once that request is merged, I'd like you to perform a back-merge from the `master` branch into this feature branch. And adjust your code accordingly. Let's discuss in person any questions you might have.